docs/jazzy.yaml

```diff
@@ -8,15 +8,11 @@ exclude:
   - "**/*Deprecations.swift"
   - "**/SwiftExtensions.swift"
 readme: 'libs/SalesforceSwiftSDK/readme.md'
-copyright: '© Copyright 2017–Current salesforce.com, inc. All rights reserved. Salesforce is a registered trademark of salesforce.com, inc.,
+copyright: '© Copyright 2017–current salesforce.com, inc. All rights reserved. Salesforce is a registered trademark of salesforce.com, inc.,
 as are other names and marks. Other marks appearing herein may be trademarks of their respective owners.'
-
-swift_version: 4.0
-
+swift_version: 4.0.3
 skip_undocumented: true
-
 theme: fullwidth
-
 clean: false
-
-xcodebuild_arguments: [-configuration, Release,-sdk, iphonesimulator,-scheme, SalesforceSwiftSDK, -workspace, SalesforceMobileSDK.xcworkspace]
+xcodebuild_arguments: [-configuration, Release, -sdk, iphonesimulator, -scheme, SalesforceSwiftSDK, -workspace, SalesforceMobileSDK.xcworkspace]
+output: ../build/artifacts/doc/SalesforceSwiftSDK
```

Comment:
Adding output directory argument to be consistent with where the other library docs reside.